### DISCUSSION TITLE
Using resizeable panels

### DISCUSSION THREAD

**Discussion by LiamCrypto** [solodit/feedback#47] [New Feature Requests]
I think it'd be more beneficial to resize the panels on the findings page rather than having to cascade them and reopen them to use the functionality. Something along the lines of react resizeable panels and potentially using a combination of the both

**Comment by hans-cyfrin** [comment]
Thanks for the thoughts.
Resizable panels are what we had in the initial versions, and we believe the current collapse/expand UI is preferred by most users.

**Reply by LiamCrypto** [reply]
Ahh thats cool, thanks for the response
